docs/pages/web/popovereducational.js

```diff
@@ -0,0 +1,628 @@
+
+      <MainSection name="Usage guidelines">
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            title="When to use"
+            description={`
+- Bringing attention to specific user interface elements for educational  or onboarding purposes.
+        `}
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="don't"
+            title="When not to use"
+            description={`
+- Displaying critical information that prevents users from accomplishing a task.
+- Displaying information out of context.
+- As a replacement for [Tooltip](/web/tooltip).
+- For presenting a list of actions or options. Use [Dropdown](/web/dropdown) instead.
+        `}
+          />
+        </MainSection.Subsection>
+      </MainSection>
+
+      <MainSection name="Best practices">
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description="Use the PopoverEducational to educate users on a new or existing feature. Be sure to use a caret pointing to the feature. If there is more than one item, use a CTA button to move the user to the next popover."
+            defaultCode={`
+function Example() {
+  const [open, setOpen] = React.useState(false);
+  const anchorRef = React.useRef();
+
+  React.useEffect(() => {
+    setOpen(true)
+  }, []);
+
+  return (
+    <Box height="100%">
+      <TapArea ref={anchorRef} rounding={3}>
+        <Box padding={3} color="secondary" height={75} width={200} rounding={3}>
+          <Flex gap={2}>
+            <Box aria-hidden height={500} width={50}>
+              <Mask rounding={3} wash>
+                <Image
+                  alt="Image of a Spanish paella from above. Yellow rice with red peppers and shrimp on top."
+                  color="rgb(231, 186, 176)"
+                  loading="lazy"
+                  naturalHeight={1}
+                  naturalWidth={1}
+                  src="https://i.ibb.co/d2tpDss/IMG-0494.jpg"
+                />
+              </Mask>
+            </Box>
+            <Flex direction="column">
+              <Text size="100">More ideas for</Text>
+              <Text weight="bold">Food, Drinks, Snacks</Text>
+            </Flex>
+          </Flex>
+        </Box>
+      </TapArea>
+      {open && (
+        <PopoverEducational
+          id="popover-do-educate"
+          idealDirection="down"
+          anchor={anchorRef.current}
+          onDismiss={() => {}}
+          message="Tap to tag a product or press and hold to see product details"
+          primaryAction={{ text: "Next"}}
+        />
+      )}
+    </Box>
+)}`}
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="don't"
+            description="Show more than one PopoverEducational at a time. If used for onboarding, show a next button instead, to launch the next popover."
```

Comment:
Given this guidance, should we add a second popover and make the button functional?

Reply:
DONE